### PR TITLE
refactor(config-map): removed config map creation (AEROGEAR-9744)

### PR DESCRIPTION
**Pull request by craicoverflow**
https://issues.jboss.org/browse/AEROGEAR-9744

Removed the creation of a config map created for each MobileSecurityServiceApp CR as it is not needed.

## Verification

1. Replace the image in `operator.yaml` with `quay.io/ephelan/mobile-security-service-operator:AEROGEAR-9744`.
1. Deploy the operator and install the Mobile Security Service with `make install`.
2. Create an app with `make example-app/apply.`
3. Check for the app CR, eg: `oc get mobilesecurityserviceapp mobile-security-service-app`
4. Delete the app CR with `make example-app/delete`.
5. It should delete successfully.

### CODE REVIEW

File: pkg/controller/mobilesecurityserviceapp/status.go [hunk 1]
```diff
@@ -66,8 +28,8 @@ func (r *ReconcileMobileSecurityServiceApp) updateBindStatus(serviceURL string,
 	}
 
 	// Check if the ConfigMap and the App is created in the Rest Service
```
Comment (camilamacedo86) [inline pkg/controller/mobilesecurityserviceapp/status.go]:
```suggestion
```

File: Makefile [hunk 2]
```diff
@@ -9,7 +9,7 @@ APP_FILE=./cmd/manager/main.go
 BIN_DIR := $(GOPATH)/bin
 BINARY ?= mobile-security-service-operator
 IMAGE_REGISTRY=quay.io
-REGISTRY_ORG=aerogear
+REGISTRY_ORG=ephelan
```
Comment (camilamacedo86) [inline Makefile]: @craicoverflow I think it was committed by mistake.

Comment (craicoverflow) [reply Makefile]: yep it was

**Comment by craicoverflow** [comment]
Hi @camilamacedo86 could you review this again?

**Pull request merged**